core_plugins/README.md

```diff
@@ -0,0 +1,30 @@
+# Kibana Core Plugins
+
+This folder contains all the Kibana core plugins. Each plugin is built on its
+own, so it contains a separate `package.json` and build script, including a
+separate TypeScript config — i.e. they are independent.
+
+TODO: describe dev setup or link to eventual package we build that handles it
+
+## Why are core plugins "independent"?
+
+There are several reasons why we want to keep core plugins independent, e.g.:
+
+- It forces us to treat internal and external plugins the same. They get the
+  same access to Kibana apis, they get the same build treatment, etc. That means
+  we treat external plugins as a true first-class citizen.
+- Plugins can't import modules in the Kibana platform, e.g.
```

Comment:
__optional nit:__ maybe that's just me, but `import in` still sounds weird, what about `import from`?

Reply:
I would also go with `import from`

Reply:
Agreed ✅